servers/curated-corpus-api/schema-admin.graphql

```diff
@@ -573,6 +573,10 @@ type Section {
   """
   active: Boolean!
   """
+  An array of active and in-active SectionItems in a Section.
+  """
+  sectionItems: [SectionItem]
```

Comment:
i think this could be `[SectionItem!]!`, right? the list is guaranteed to return. it might be empty, but cannot contain `null` values.

Reply:
hmm when creating a new Section, `sectionItems` is `null` & a non-nullable field cannot return `null`

Reply:
@jpetto the graph returns an internal server error as [SectionItem!]! expects an array but the prisma db returns null. I will modify the Section mutations to return an empty array instead of null when creating/updating a section.